### PR TITLE
Don't release stock if at 0.

### PR DESCRIPTION
**Pull request by notbakaneko**

### CODE REVIEW

File: app/Models/Store/Product.php [hunk 1]
```diff
@@ -169,6 +169,12 @@ public function release($quantity)
             return true;
         }
 
+        // stock may have been directly updated to 0.
+        // TODO: should count reservations and available stock separately or something.
+        if ($this->stock <= 0) {
+            return false;
```
Comment (nanaya) [inline app/Models/Store/Product.php]: wtf does the return do ಠ_ಠ (also in the one above)

Comment (nanaya) [reply app/Models/Store/Product.php]: *return value

Comment (notbakaneko) [reply app/Models/Store/Product.php]: SUCCE-..wait the last one doesn't return 😕

Comment (nanaya) [reply app/Models/Store/Product.php]: looks like the original function returned the save return and then nuked when changed to use increment.

**Pull request merged**